client/src/components/OnePage/Footer/Footer.tsx

```diff
@@ -1,53 +1,132 @@
+    marginTop: '0',
+    width: 'auto',
+    fontSize: '18px',
+    fontWeight: 350,
+  },
+}));
+
+const GroupsBox = styled(Box)(({ theme }) => ({
+  alignItems: 'flex-start',
   display: 'flex',
-  color: 'white',
-  flexDirection: 'column',
-  justifyContent: 'center',
-  alignItems: 'center',
-  textAlign: 'center',
-  maxWidth: 895,
+  justifyContent: 'flex-start',
   [theme.breakpoints.down('md')]: {
-    marginTop: '150px',
+    width: 'auto',
   },
 }));
 
-const FooterEndLine = styled(Box)(() => ({
+const CustomBoxGroup = styled(Box)(({ theme }) => ({
+  width: '190px',
+  marginRight: '40px',
+  [theme.breakpoints.down('md')]: {
+    marginRight: '24px',
+    width: 'auto',
+    fontSize: '14px',
+    fontWeight: 500,
+    marginBottom: '10px',
+  },
+}));
+
+const CopyrightContainer = styled(Container)(({ theme }) => ({
   alignItems: 'center',
-  textAlign: 'center',
+  display: 'flex',
+  justifyContent: 'space-between',
+  marginBottom: '20px',
+  marginTop: '20px',
+  color: '#FF6700',
+  fontWeight: 'bold',
+  [theme.breakpoints.down('md')]: {
+    alignItems: 'flex-start',
+    flexDirection: 'column',
+    fontSize: '12px',
+    fontWeight: 350,
+    justifyContent: 'flex-start',
+    marginBottom: '35px',
+    marginTop: '20px',
+  },
+}));
+
+const CustomTermsBox = styled(Box)(({ theme }) => ({
+  display: 'flex',
+  [theme.breakpoints.down('md')]: {
+    fontSize: '0.75rem',
+  },
+}));
+
+const CustomTermsTypography = styled('p')(({ theme }) => ({
+  marginLeft: '20px',
+  [theme.breakpoints.down('md')]: {
+    margin: '4px 8px 0 0',
+  },
 }));
 
 const Footer = () => {
   return (
-    <FooterContainer
-      maxWidth={false}
-      disableGutters
-      className="container full-width"
+    <Container
+      className="dark-container"
+      sx={{ paddingTop: '120px', paddingBottom: '10px', position: 'relative' }}
     >
-      <FooterContent className="footer-content">
-        <FooterEndLine className="footer-end-items">
-          <Typography>Copyright © 2023 Chattie</Typography>
-          <Typography>Created by Kinga Kuś & Adam Kostuch</Typography>
-        </FooterEndLine>
-      </FooterContent>
-    </FooterContainer>
+      <CustomContainer className="footerContainer">
+        <CustomBoxItems className="footer-items">
+          <SloganBox className="slogan">
+            <h3>
+              Hello. We are CHATTIE,
+              <br />
+              connecting people with others
+              <br />
+              and with robots.
+            </h3>
+          </SloganBox>
+          <GroupsBox className="groups">
+            {FooterData.map((data) => (
+              <CustomBoxGroup className={data.className} key="">
+                <h4>{data.name}</h4>
+                <div style={{ fontSize: '25px' }}>
+                  {data.subnames.map((subname) => (
+                    <p key="subname">{subname}</p>
+                  ))}
+                </div>
+              </CustomBoxGroup>
+            ))}
```

Comment:
Key should be unique for every row that is being rendered. So in this case instead of [`'subname'`, `'subname'`, `'subname'`, ...], you want to specify it more like this: [`{subname}`, `{subname}`, `{subname}`, ...] or even better: [`{subname + idx}`, `{subname + idx}`, `{subname + idx}`, ....]:

```suggestion
            {FooterData.map((data, idx) => (
              <CustomBoxGroup className={data.className} key={data.name + idx}>
                <h4>{data.name}</h4>
                <div style={{ fontSize: '25px' }}>
                  {data.subnames.map((subname, idx) => (
                    <p key={subname + idx}>{subname}</p>
                  ))}
                </div>
              </CustomBoxGroup>
            ))}
```

Reply:
Okie, I will change it!